.github/workflows/build.yml

```diff
@@ -141,8 +142,13 @@ jobs:
         if: ${{ inputs.build_debug }}
         working-directory: thingy91x-oob/app
         run: |
-          echo CONFIG_APP_MEMFAULT_UPLOAD_METRICS_ON_CLOUD_READY=y >> overlay-memfault.conf
-          west build -p -b thingy91x/nrf9151/ns -p --sysbuild -- -DEXTRA_CONF_FILE="overlay-memfault.conf;overlay-modemtrace.conf;overlay-etb.conf"
+          cp overlay-memfault.conf overlay-memfault-debug.conf
```

Comment:
Copy the original config and set necessary configs again to avoid hidden dependency to the previous build.